pyLibrary/queries/expressions.py

```diff
@@ -2695,12 +2695,12 @@ def to_sql(self, schema, not_null=False, boolean=False):
                 if els_[t] == None:
                     pass
                 else:
-                    output[t] = "CASE WHEN " + when.b + " THEN NULL ELSE " + els_[t] + " END"
+                    output[t] = "CASE WHEN " + unwrap(when).values()[0] + " THEN NULL ELSE " + els_[t] + " END"
```

Comment:
as per our call, do not `unwrap(when).values()` you will lose type safety; update the function that provided `when` so it will return a boolean.

please make a new branch!  this pull requests keeps changing.